src/backend/src/controllers/projects.controllers.ts

```diff
@@ -299,3 +299,54 @@ export const editProject = async (req: Request, res: Response) => {
   // return the updated work package
   return res.status(200).json(updatedProject);
 };
+
+export const setProjectTeam = async (req: Request, res: Response) => {
+  const { body } = req;
+
+  // check for valid WBS number
+  const parsedWbs: WbsNumber = validateWBS(req.params.wbsNum);
+
+  if (!isProject(parsedWbs)) {
+    return res.status(404).json({ message: `${req.params.wbsNum} is not a valid project WBS #!` });
+  }
+
+  const wbsEle = await prisma.wBS_Element.findUnique({
+    where: {
+      wbsNumber: {
+        carNumber: parsedWbs.carNumber,
+        projectNumber: parsedWbs.projectNumber,
+        workPackageNumber: parsedWbs.workPackageNumber
+      }
+    },
+    include: {
+      project: true
+    }
+  });
+
+  if (wbsEle === null) {
+    return res.status(404).json({ message: `project ${req.params.wbsNum} not found!` });
+  }
+
+  // find the associated project
+  const projectID = wbsEle.project?.projectId;
+
+  // check for valid team
+  const team = await prisma.team.findUnique({ where: { teamId: body.teamId } });
+  if (!team) {
+    return res.status(404).json({ message: `team with id ${body.teamId} not found.` });
+  }
+  // check for user and user permission
+  const user = await prisma.user.findUnique({ where: { userId: body.submitterId } });
+  if (!user) {
+    return res.status(404).json({ message: `user with id #${body.submitterId} not found` });
+  }
+  if (user.role === Role.GUEST || user.role === Role.MEMBER) return res.status(403).json({ message: 'Access Denied' });
```

Comment:
only people allowed to do this are: APP_ADMIN, ADMIN, and the team lead of the team (so something similar to team.teamLeadId === submitterId)